app.php

```diff
@@ -0,0 +1,213 @@
+use Appwrite\Services\Storage;
+use Appwrite\Services\Users;
+
+
+$endPoint = 'https://localhost/v1';
+$projectId = '<Your Project ID />';
+$apiKey = '<Your Project Secret Api key/>';
+
+$client = new Client();
+
+$client->setEndpoint($endPoint);
+$client->setProject($projectId);
+$client->setKey($apiKey);
+$collectionId = $userId = 0;
+$dataBase = new Database($client);
+$storage = new Storage($client);
+$users = new Users($client);
+
+# API Calls
+#   - api.createCollection
+#   - api.listCollection
+#   - api.addDoc
+#   - api.listDoc
+#   - api.uploadFile
+#   - api.listFiles
+#   - api.deleteFile
+#   - api.createUser
+#   - api.listUser
+
+# List of API definitions
+
+/**
+ * @throws Exception
+ */
+function createCollection()
+{
+    # code...to create collection
+    global $collectionId, $dataBase;
+
+    $response = $dataBase->createCollection(
+        'Movies',
+        ['*'],
+        ['*'],
+        [
+            [
+                'label' => "Name",
+                'key' => "name",
+                'type' => "text",
+                'default' => "Empty Name",
+                'required' => true,
+                'array' => false
+            ],
+            [
+                'label' => 'release_year',
+                'key' => 'release_year',
+                'type' => 'numeric',
+                'default' => 1970,
+                'required' => true,
+                'array' => false
+            ]
+        ]
+    );
+    $collectionId = $response['id'];
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function listCollection()
+{
+    global $dataBase;
+
+    echo "Running List Collection API";
+    $response = $dataBase->listCollections();
+    $collection = $response['$collection'];
+    var_dump($collection);
+}
+
+/**
+ * @throws Exception
+ */
+function addDoc()
+{
+    global $collectionId;
+    global $dataBase;
+
+    $collectionId = "0";
+    echo "Running Add Document API";
+    $response = $dataBase->createDocument(
+        $collectionId,
+        [
+            'name' => "Spider Man",
+            'release_year' => '1920',
+        ],
+        ['*'],
+        ['*']
+    );
+
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function uploadFiles()
+{
+    global $storage;
+
+    $fileName = 'test.txt';
+    echo "Running upload file API";
+    $response = $storage->createFile(
+        curl_file_create($fileName),
+        [],
+        []
+    );
+
+    var_dump($response);
+}
+
+/**
+ * @throws Exception
+ */
+function listFiles()
+{
+    global $storage;
+
+    echo "Running List Files API";
+    $result = $storage->listFiles();
+    $fileCount = $result['sum'];
+    $files = $result['files'];
+    var_dump($fileCount, $files);
+}
+
+/**
+ * @throws Exception
+ */
+function deleteFile()
+{
+    global $storage;
+
+    echo "Running Delete File API";
+    $result = $storage->listFiles();
+    $firstFileId = 'test.txt';
+    //$first_file_id = $result['files'][0]['$id'];
```

Comment:
Is this correct? If yes, remove this comment please.

Reply:
removed